modules/apps/headless/headless-admin-user/headless-admin-user-impl/rest-openapi.yaml

```diff
@@ -459,7 +459,7 @@ components:
                     description:
                         The segment's criteria.
                     readOnly: true
-                    type: string
+                    type: object
```

Comment:
Better to make the OpenAPI change in one commit and commit the autogenerated changes in the next to easily understand the changes in the API definition